CHANGES

```diff
@@ -23,6 +23,9 @@ Changes
 
 Enhancements
 
+* convert figure components to SVG and save as scalable PDFs (#243)
+* add RDKit mol object to dihedrals plot with dihedral atom indices
+  labeled and dihedral atom group bonds highlighted (#243)
```

Comment:
need to specify where — a user should be able to read CHANGES and figure out if anything affects them. You have to say in which module this and what the context is. Don't make people go to the issues.